snippets/overview.php

```diff
@@ -124,6 +124,9 @@
 $table = new \theme_boost_union\table\snippets_overview();
 $table->define_baseurl($PAGE->url);
 
+// Show alert if the Boost Union theme is not active.
```

Comment:
If you change `theme_boost_union_is_not_active_alert()` as requested above, you will see that the notification is not shown anymore on this page where you want it to appear.

You should move the call to `theme_boost_union_is_not_active_alert()` below the call to echo `$OUTPUT->header();` to solve this.